src/nodes/CronExpressionInput.js

```diff
@@ -131,13 +131,33 @@ export class CronExpressionInput extends CronComponent {
             }
         });
 
-        this.getElements('.propagationClass').forEach((element) =>
-            element.addEventListener('input', (e) => e.stopPropagation())
-        );
+        this.getElements('.propagationClass').forEach(function (element) {
```

Comment:
Is it better to use classic fonctions there? I know the codestyle is not really set but there are more arrow functions in this file.